src/appleseed/renderer/kernel/lighting/lighttree.cpp

```diff
@@ -247,8 +247,8 @@ float LightTree::recursive_node_update(size_t node_index, size_t node_level)
 }
 
 std::pair<size_t, float> LightTree::sample(
-    const foundation::Vector3d    surface_point,
-    float                         s) const
+    const foundation::Vector3d&     surface_point,
+    float                           s) const
```

Comment:
The spacing between the first argument and it's type should be  multiple of 4 or the character column? (I tend to stick to column).